src/cardboard-distorter.js

```diff
@@ -98,8 +98,7 @@ function createRenderTarget(renderer) {
   var parameters = {minFilter: THREE.LinearFilter,
                     magFilter: THREE.LinearFilter,
                     format: THREE.RGBFormat,
-                    stencilBuffer: false,
-                    depthBuffer: false};
```

Comment:
Fixed in https://github.com/borismus/webvr-boilerplate/commit/f48f016dee7833fcf1522157e77913167f16cd7a